config/overlays/release/default_patch.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: gateway-plugins
+  namespace: aibrix-system
+spec:
+  template:
+    spec:
+      containers:
+        - name: gateway-plugin
+          resources:
+            limits:
+              cpu: 2
```

Comment:
BTW a minor question, just curious does 2C/4G work based on your experiment? such memory heavy (1:4) configuration is not that common.